be/app/Http/Controllers/UserController.php

```diff
@@ -17,7 +17,7 @@ class UserController extends Controller
      */
     public function index()
     {
-        $users = User::all();
+        $users = User::all()->except(1);
```

Comment:
- what is this for? this is very strange :sweat_smile:

Reply:
> hi sir about the except, I reserved the first ID po for admin.

oki. but admin is still a user. Ahh Please revert the changes.
just add a seeder for admin and assign id to 1.

Reply:
>hi sir about the except, I reserved the first ID po for admin.

oki. but admin is a user as well. Please revert the changes.
just generate a new seeder for admin and assign `id` to 1